Update README

felipecrs opened the pull request:
@alexxit, I went through the README and made some updates:

1. Renamed Hass to Home Assistant, as per Home Assistant project preferences ([I did the same in Frigate](https://github.com/blakeblackshear/frigate/pull/16337))
2. Remove references to Lovelace, as Home Assistant project prefers to call it Dashboard
3. Update some links like Frigate Card -> Advanced Camera Card
4. Update config dir example to `/homeassistant` which is what more modern add-ons will use (like Frigate 0.16)
5. Some other minor stuff

I hope you like it.

felipecrs commented on README.md:
```diff
@@ -329,8 +329,8 @@ streams:
 
-- FFmpeg preistalled for **Docker** and **Hass Add-on** users
-- **Hass Add-on** users can target files from [/media](https://www.home-assistant.io/more-info/local-media/setup-media/) folder
+- FFmpeg preistalled for **Docker** and **Home Assistant Add-on** users
```
```suggestion
- FFmpeg preinstalled for **Docker** and **Home Assistant Add-on** users
```

AlexxIT commented:
1. It's better not to change source and module name. Because this name used in config.
2. In other places (ex. Home Assistant Add-on) are fine
3. I'm not sure about changes path to config dir `/homeassistant`. This should be tested carefully.

felipecrs commented:
@AlexxIT, thanks a lot for looking into it.

---

1. **Note I did not rename the YAML examples itself, they still point to `hass:`**. I can revert if you want, but I think it looks more professional this way. Just say the word.

---

3. I will revert this, we can discuss it in the future.

felipecrs commented:
> 1. It's better not to change source and module name. Because this name used in config.

@AlexxIT I reverted this, and fixed all conflicts caused by https://github.com/AlexxIT/go2rtc/pull/1773.

Also, I refactored all occurrences of _2-way_ and _two way_ terminology into _two-way_. Also refactored _Add-on_ into _add-on_.

I believe all your concerns are sorted now.

felipecrs commented:
@ehn maybe you can help review this PR too. :)

felipecrs commented on README.md:
````diff
@@ -1134,9 +1144,9 @@ streams:
 ```
 
-**PS.** Default Home Assistant lovelace cards don't support two-way audio. You can use 2-way audio from [Add-on Web UI](https://my.home-assistant.io/redirect/supervisor_addon/?addon=a889bffc_go2rtc&repository_url=https%3A%2F%2Fgithub.com%2FAlexxIT%2Fhassio-addons), but you need to use HTTPS to access the microphone. This is a browser restriction and cannot be avoided.
+**PS.** Default Home Assistant cards don't support two-way audio. You can use two-way audio from the [add-on Web UI](https://my.home-assistant.io/redirect/supervisor_addon/?addon=a889bffc_go2rtc&repository_url=https%3A%2F%2Fgithub.com%2FAlexxIT%2Fsource-hassio-addons), but you need to use HTTPS to access the microphone. This is a browser restriction and cannot be avoided.
````
```suggestion
**PS.** Default Home Assistant cards don't support two-way audio. You can use two-way audio from the [add-on Web UI](https://my.home-assistant.io/redirect/supervisor_addon/?addon=a889bffc_go2rtc&repository_url=https%3A%2F%2Fgithub.com%2FAlexxIT%2Fhassio-addons), but you need to use HTTPS to access the microphone. This is a browser restriction and cannot be avoided.
```

ehn commented:
> @ehn maybe you can help review this PR too. :)

Looks good to me. :)

ehn reviewed (approved):
This looks good to me.

felipecrs commented:
That's ok. Thanks a lot for merging!
